homeassistant/components/xiaomi_ble/manifest.json

```diff
@@ -24,5 +24,5 @@
   "dependencies": ["bluetooth_adapters"],
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_ble",
   "iot_class": "local_push",
-  "requirements": ["xiaomi-ble==0.30.2"]
+  "requirements": ["xiaomi-ble==0.31.1"]
```

Comment:
Usually we would bump the library in a different PR, but in this case its likely that its going to raise KeyError as soon as you set up the sensor since the mappings are otherwise missing and its small enough to do it in the same PR.